docs/pages/custom-builds/schema.mdx

```diff
@@ -1300,6 +1300,98 @@ build:
+| `maestro_version` | **string** Maestro version to install (e.g. 1.35.0). If not provided, `install_maestro` will install the latest version. |
+
+<BoxLink
+  title="eas/install_maestro source code"
+  description="View the source code for the eas/install_maestro function on GitHub."
+  Icon={GithubIcon}
+  href="https://github.com/expo/eas-build/blob/main/packages/build-tools/src/steps/functions/installMaestro.ts"
+/>
+
+#### `eas/start_android_emulator`
+
+Starts an Android emulator you can use to test your apps on. Only available when running a build for Android.
+
+```yaml build-and-test.yml
+build:
+  name: Build and test
+  steps:
+    - eas/build
+    # @info #
+    - eas/start_android_emulator:
+        inputs:
+          system_image_package: system-images;android-30;default;x86_64
+    # @end #
+    # ... Maestro setup and tests
+```
+
+| Input                  | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               |
+|------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `device_name`          | **string** Name for the created device. You may wish to customize it if starting multiple emulators.                                                                                                                                                                                                                                                                                                                                                                                                      |
+| `system_image_package` | **string** Android package path you wish to use for the emulator, e.g. `system-images;android-30;default;x86_64`.<br />To get list of available system images, run `sdkmanager --list` on your computer ([command docs](https://developer.android.com/tools/sdkmanager#list)). VMs run on x86_64 architecture, so always choose `x86_64` package variants. The `sdkmanager` tool comes from Android SDK command-line tools ([installation instructions](https://developer.android.com/tools/sdkmanager)). |
```

Comment:
```suggestion
| `system_image_package` | **string** Android package path to use for the emulator. For example, `system-images;android-30;default;x86_64`.<br />To get a list of available system images, run [`sdkmanager --list`](https://developer.android.com/tools/sdkmanager#list) on a local computer. VMs run on x86_64 architecture, so always choose `x86_64` package variants. The [`sdkmanager` tool](https://developer.android.com/tools/sdkmanager) comes from Android SDK command-line tools. |
```

- Let's highlight sdkmanager command and sdkmanager tools as links instead of explicitly specifying those docs links.